homeassistant/components/ozw/cover.py

```diff
@@ -22,6 +23,7 @@
 
 SUPPORTED_FEATURES_POSITION = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
 SUPPORT_GARAGE = SUPPORT_OPEN | SUPPORT_CLOSE
+VALUE_ID = "Value"
```

Comment:
This constant isn't used in this module. Please move it to the test module.

Reply:
Done